SUMMARY.md

```diff
@@ -9,30 +9,31 @@
 * [Pythnet Price Feeds](pythnet-price-feeds/pythnet-price-feeds.md)
   * [On-Demand Updates](pythnet-price-feeds/on-demand.md)
   * [Using Price Feeds](pythnet-price-feeds/best-practices.md)
-  * [Price Service](pythnet-price-feeds/price-service.md)
-  * [Pyth Cross-chain Batches](pythnet-price-feeds/cross-chain-batches.md)
   * [Pyth on EVM](pythnet-price-feeds/evm.md)
   * [Pyth on Aptos](pythnet-price-feeds/aptos.md)  
   * [Pyth on BAS Chains](pythnet-price-feeds/bas.md) 
   * [Pyth for Off-chain Apps](pythnet-price-feeds/off-chain.md)
+  * [Price Service](pythnet-price-feeds/price-service.md)
+  * [Price Feed Batches](pythnet-price-feeds/cross-chain-batches.md)  
 
 * [Publish Data](publishers/publish-data.md)
   * [Understanding Publishing Slots](publishers/understanding-publishing-slots.md)
   * [pyth-client Websocket API](publishers/pyth-client-websocket-api.md)
   * [Confidence Interval and Crypto Exchange Fees](publishers/confidence-interval-and-crypto-exchange-fees.md)
 
-* [How Pyth Works](how-pyth-works/how-pyth-works.md)
-  * [Design Overview](how-pyth-works/design-overview.md)
+* [How Pyth Works](how-pyth-works/design-overview.md)
   * [Pythnet](how-pyth-works/pythnet.md)
   * [Cross-chain](how-pyth-works/cross-chain.md)
-  * [Solana Account Structure](how-pyth-works/account-structure.md)
-  * [Product Metadata](how-pyth-works/product-metadata.md)
+  * [Oracle Program](how-pyth-works/oracle-program.md)
+  * [Account Structure](how-pyth-works/account-structure.md)
+  * [Product Metadata](how-pyth-works/product-metadata.md)  
   * [Price Aggregation](how-pyth-works/price-aggregation.md)
   * [EMA Price Aggregation](how-pyth-works/ema-price-aggregation.md)
-  * [Accounts](https://pyth.network/developers/accounts/)
-  * [Network Participants](how-pyth-works/network-participants.md)
-  * [Network Interactions](how-pyth-works/network-interactions.md)
-  * [Participant Incentives](how-pyth-works/participant-incentives.md)
+
+* [Whitepaper](whitepaper/whitepaper.md)
```

Comment:
the old section was a mix of how things currently work and stuff described in the whitepaper that doesn't exist. separate the two.

Reply:
the whitepaper section right now is all copy-paste from the existing text. i'll circle back and update that text in a follow up pr